participants/djg.json

```diff
@@ -0,0 +1,45 @@
+// Please provide your info in your own .json file.
+// See https://jscraftcamp.org/registration for more information
+{
+	// your real name (required by location host)
+	"realName": {
+		"givenName": "Dirk",
+		"familyName": "Gomez"
+	},
+	// please put in the account name of the PR creator, if you sign up somebody else
+	"githubAccountName": "dirkgomez",
+	// either both days or at least one day has to be set to true
+	"when": {
+		// June 7th, 2024
+		"friday": true,
+		// June 8th, 2024
+		"saturday": false
+	},
+	// if you are willing to take session notes and publish them to github (required)
+	"iCanTakeNotesDuringSessions": false,
+	// your current interests (JS and in general) (required)
+	"tags": ["TypeScript", "Serverless", "AWS"],
+	// if you only eat vegan food (optional)
+	"vegan": false,
+	// if you only eat vegan or vegetarian food (optional)
+	"vegetarian": false,
+	// what you cannot eat or drink (optional); If you don't want to put it in here, message the organizers.
+	// IMPORTANT: we cannot guarantee that food for every diet will be available,
+	//            if you have gluten free diet, please make backup plans.
+	"allergies": [],
+	// tell us a few words how JavaScript affects you (required)
+	"whatIsMyConnectionToJavascript": "I started with JavaScript when it was considered a toy language where you simply cut and paste code and last coded JavaScript with jQuery. These days I mostly use TypeScript for IAAC.",
+	// what can you contribute to the bar camp (required)
+	"whatCanIContribute": "TypeScript in AWS CSK.",
```

Comment:
I only know AWS CDK - typo or is this another thing? 😅 I always get confused by abbreviations...